.github/workflows/langchain_ci.yml

```diff
@@ -84,6 +84,13 @@ jobs:
       - name: Run extended tests
         run: make extended_tests
 
+      - name: Run TileDB tests
+        run: |
+          sudo apt install libopenblas-dev
+          poetry install --with=test_integration
+          poetry run pip install --no-cache-dir "tiledb-vector-search@git+https://github.com/TileDB-Inc/TileDB-Vector-Search#egg=tiledb-vector-search&subdirectory=apis/python"
```

Comment:
If we are going to upstream this CI change, we should pin a version here IMO

Reply:
Now that 0.0.18 is released I changed this to not build from github.
I am not plan to upstream this CI change. It is more about testing our changes in this repo.